test/colonel_kurtz_ex_test.exs

```diff
@@ -1,4 +1,41 @@
 defmodule ColonelKurtzTest do
```

Comment:
Eventually this will get broken out more into logical bits, but for the time being I just wanted some smoke tests that would let me do a couple trickier refactors.